helm/cosmo/charts/controlplane/values.yaml

```diff
@@ -202,7 +202,7 @@ configuration:
     # -- The HTTP path where metrics are exposed. Default is "/metrics".
     path: "/metrics"
     # Use this section to disable/enable pod monitoring over [GCP managed collection](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#config-mgd-collection).
-    gcp_monitoring:
+    gcpMonitoring:
```

Comment:
I'm with it but this a breaking and we need to consider this when updating our cloud values.